packages/react-components/react-rating-preview/stories/RatingDisplay/index.stories.tsx

```diff
@@ -4,6 +4,11 @@ import descriptionMd from './RatingDisplayDescription.md';
 import bestPracticesMd from './RatingDisplayBestPractices.md';
 
 export { Default } from './RatingDisplayDefault.stories';
+export { Color } from './RatingDisplayColor.stories';
+export { Max } from './RatingDisplayMax.stories';
+export { Step } from './RatingDisplayStep.stories';
+export { Size } from './RatingDisplaySize.stories';
+export { Shape } from './RatingDisplayShape.stories';
```

Comment:
These don't need to be in alphabetical order. It's better to sort them from basic to advanced. For example:
* Value (recommend adding)
* Count (recommend adding)
* Max
* Size
* Color
* Shape